src/transformers/utils/documentation_tests.txt

```diff
@@ -220,4 +220,5 @@ src/transformers/models/yolos/configuration_yolos.py
 src/transformers/models/yolos/modeling_yolos.py
 src/transformers/models/x_clip/modeling_x_clip.py
 src/transformers/models/yoso/configuration_yoso.py
+src/transformers/models/timesformer/configuration_timesformer.py
```

Comment:
Here is our commit in the `utils/documentations_test.txt` file to address the timesformer_configuration

Reply:
This is not the file `utils/documentations_test.txt`, this is `src/transformers/utils/documentation_tests.txt`, which is newly added in this PR, but it should not be added.